package.json

```diff
@@ -51,6 +51,7 @@
     "node-libs-browser": "^1.0.0",
     "node-sass": "^3.2.0",
     "postcss-loader": "^0.13.0",
+    "prop-types": "^15.5.8",
```

Comment:
This should probably go in `dependencies` as per https://github.com/reactjs/prop-types/blob/master/README.md#how-to-depend-on-this-package